act-rules-format.bs

```diff
@@ -15,21 +15,21 @@ Markup Shorthands: markdown yes
 Introduction {#intro}
 =====================
 
-There are currently many tools available which aid their users in testing web content for conformance to accessibility standards such as the Web Content Accessibility Guidelines (WCAG). As the web develops in both size and complexity, these tools are essential for managing the resources available on the web.
+There are currently many tools available which aid their users in testing web content for conformance to accessibility standards such as the [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/) [[WCAG20]]. As the web develops in both size and complexity, these tools are essential for managing the resources available on the web.
```

Comment:
I recommend dropping the version number "2.0", and to use this link to the overview for WCAG instead: https://www.w3.org/WAI/intro/wcag

Reply:
Hi Shadi, per Stein Erik's recommendation I used the non-versioned TR link, https://www.w3.org/TR/WCAG/

Reply:
As per 5/31 meeting, updating to overview page, https://www.w3.org/WAI/standards-guidelines/wcag/